lib/packer/installer.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'open3'
+require 'digest/sha1'
+
+module Packer
+  class Installer
+    delegate :logger, :config, to: :@packer
+
+    def initialize(packer)
+      @packer = packer
+    end
+
+    def install
```

Comment:
```
return true if fresh?

record_package_digest
run_npm_install.tap do |success|
   remove_package_digest unless success
 end
```

perhaps?

Reply:
Fixed in https://github.com/simplybusiness/packer/pull/6/commits/4a13bf3f066b2e1741091dedfb37d44c8d6d6a44